test/espnet2/utils/test_plot_sinc_filters.py

```diff
@@ -0,0 +1,44 @@
+import matplotlib
+import os
+import torch
+
+import egs2.TEMPLATE.asr1.pyscripts.utils.plot_sinc_filters as psf
+from espnet2.layers.sinc_conv import SincConv
```

Comment:
We don't have importing system for pyscripts, so could you change this file to a script in `test_utils`?